conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureHandler.java

```diff
@@ -128,7 +152,23 @@ public Builder fallback(HttpHandler value) {
 
         public HttpHandler build() {
             checkOverlappingPaths();
-            return new ConjureHandler(fallback, endpoints);
+            EndpointHandlerWrapper current = endpoint -> Optional.empty();
+            current = stackEndpointHandlerWrapper(current, WRAPPERS_AFTER_BLOCKING.reverse());
+            current = stackEndpointHandlerWrapper(current, wrappersJustBeforeBlocking);
+            current = stackEndpointHandlerWrapper(current, WRAPPERS_BEFORE_BLOCKING.reverse());
```

Comment:
Can we inline creation of a single List<EndpointHandlerWrapper> in this method, removing both `WRAPPERS_AFTER_BLOCKING` and `WRAPPERS_BEFORE_BLOCKING`? That way I think it will be easier to follow the order of operations.

Reply:
done